### ISSUE TITLE
Terminal chevron not accessible

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#122329]
Testing #122242

Move keyboard focus to terminal title area. The chevron is not keyboard focusable, thus it can not be triggered via keyboard.
I would expect that left - right could focus onto it.

As inspiration you can check how the debug dropdown + button handle left right focus.

![Screenshot 2021-04-27 at 15 02 52](https://user-images.githubusercontent.com/1926584/116246092-f0e6e300-a769-11eb-9d04-3248c13ec43b.png)

**Comment by isidorn** [comment]
Works great -> verified
